applications/openshift/networking/default_ingress_ca_replaced/rule.yml

```diff
@@ -0,0 +1,47 @@
+prodtype: ocp4
+
+title: 'Ensure that the default Ingress CA (wildcard issuer) has been replaced'
+
+description: |-
+  Check that the default Ingress CA has been replaced.
+
+rationale: |-
+  OpenShift auto-generates several PKIs to serve TLS on different
+  endpoints of the system. It is possible and necessary to configure a
+  custom PKI which allows external clients to trust the endpoints.
+
+  The Ingress Operator is the component responsible for enabling external
+  access to OpenShift Container Platform cluster services. The aforementioned
+  operator creates an internal CA and issues a wildcard certificate that is
+  valid for applications under the .apps sub-domain. Both the web console
+  and CLI use this certificate as well. The certificate and key would need
+  to be replaced since a certificate coming from a trusted provider is
+  needed.
+
+  <tt>https://docs.openshift.com/container-platform/latest/security/certificates/replacing-default-ingress-certificate.html</tt>
```

Comment:
There is a specific jinja template for this called `weblink`:

```
{{{ weblink(link="https://docs.oracle.com/en/operating-systems/oracle-linux/selinux/") }}}
```

You can use that instead.